simple/lint.go

```diff
@@ -1707,3 +1708,52 @@ func (c *Checker) LintRedundantSprintf(j *lint.Job) {
 		ast.Inspect(f, fn)
 	}
 }
+
+func (c *Checker) LintStringCopy(j *lint.Job) {
+	emptyStringLit := func(e ast.Expr) bool {
+		bl, ok := e.(*ast.BasicLit)
+		return ok && bl.Value == `""`
+	}
+	fn := func(node ast.Node) bool {
+		switch x := node.(type) {
+		case *ast.BinaryExpr: // "" + s, s + ""
```

Comment:
Check that the operation is `+` and not, for example, `==` – also add a test for that.